api/src/main/java/com/datastrato/gravitino/rel/expressions/Indexes/Index.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.rel.expressions.Indexes;
```

Comment:
a) why using "Indexes" as the package name?
b) why index is under expression package?